src/Assets/TestProjects/VSTestMSBuildParameters/Tests.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections;
+using System.Reflection;
+
+namespace TestNamespace
+{
+    [TestClass]
+    public class Tests
+    {
+        [TestMethod]
+        public void TestMSBuildParameters()
+        {
+            var assemblyInfoVersion = Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion;
+            Assert.AreEqual(assemblyInfoVersion, "1.2.3");
```

Comment:
Expected is the first parameter.

```suggestion
            Assert.AreEqual("1.2.3", assemblyInfoVersion);
```